tardis/montecarlo/montecarlo_configuration.py

```diff
@@ -2,4 +2,6 @@
 single_packet_seed = -1
 temporary_v_packet_bins = None
 number_of_vpackets = 0
-line_interaction_type = None
\ No newline at end of file
+montecarlo_seed = 0
```

Comment:
are these constants?

Reply:
Once they're set by the configuration file they remain constants afterward, yes